### ISSUE TITLE
RSS feed not refreshing

### ISSUE THREAD

**Issue by jmalarcon** [thepracticaldev/dev.to#646]
### Current Behavior

I'm added this URL as the origin of my posts in order to import them to Dev.to:

https://rssmixer.com/feed/jmalarcon-posts.xml

The problem is that it simply does not work. In my dashboard nothing is imported and I don't see any new posts coming into it for me to edit and publish.

I've tried to delete, save and set it again, waited a long time for it to refresh (since it's asynchronous), to no avail.

### Expected Behavior
It should import any new post that appears in the RSS contents into my dashboard.

### Mobile & Desktop Screenshots/Recordings
![image](https://user-images.githubusercontent.com/1775322/45496667-436c6000-b776-11e8-853d-9ea8edbf2b7f.png)

### Additional Info (device/browser, helpful links)
I'm using Google Chrome 69.0.3497.92 (Official Build) (64-bit) under Windows 10, fully updated.

Thanks!

**Comment by Zhao-Andy** [comment]
Hey @jmalarcon, thanks for the heads up. Looks like this issue is with your RSS feed. It's a bit hard to parse since it's not in the right format, which is a format that we could communicate better. If you can edit your feed, try having it follow a structure similar to Medium's default [here's an example](https://medium.freecodecamp.org/feed).

We'll definitely write up some documentation for formatting your RSS feed properly.

**Comment by jmalarcon** [comment]
OK, thank you. I'm using a quite well known service called [rssmixer.com](https://rssmixer.com/). I'll try to change the service and use another one :-(

Thanks!

**Comment by jmalarcon** [comment]
I've been reading the [FAQ for RSSMixer](https://rssmixer.com/faq) and they generate the feed in Atom format (which in fact is pretty common). I guess DEV doesn't support Atom and just RSS. I thik it's probably a good idea to make a note about it in the settings page.

HTH.

**Comment by Zhao-Andy** [comment]
I believe the gem we use (FeedJira) supports Atom feeds, and that there may have been a blip with your last feed format. I'm looking at your feed now and it looks like it has a format similar to a Medium's feed, but the service is still RSSMixer. Are your posts pulling in with the updated feed?

**Comment by jmalarcon** [comment]
Hi:

Thanks for the update. In fact I've switched to http://www.rssmix.com wich is a different RSS-mixing service that returns RSS and not Atom, so It's working but it's not the original feed.

I'll change back to the previous one and as soon as I publish something new will let you know if it's working...

**Comment by Dhruv-Garg79** [comment]
I am facing the same issue.
When data is updated in RSS Feed, it does not reflect in dev.to posts. 
So I deleted all the posts imported from RSS feed, and now even after trying to reload, nothing is coming
